### PR TITLE
Add styling and specs

### PR DESCRIPTION
**Pull request by BranBayou**
### In this project I implemented the following requirements 
Interactions
Splash screen
A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (categories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
Testing requirements
Create unit and integration tests for all the most important components of your RoR application.
Screenshot from 2023-11-24 11-47-08

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +56,10 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## Video Presentaion <a href="https://www.loom.com/share/61f331921b974c76b0bfd2b6e51b8209?sid=5fa316cc-7bad-4a45-be69-aa8115efdfa3" name="live-demo">video-link</a>
+
+## Live Demo <a href="https://my-budget-hq38.onrender.com" name="live-demo-link">Live demo link</a>
+
 <!-- GETTING STARTED -->
```
Comment (OLIPLICHE) [inline README.md:63]: - Hello, can you "acknowledge" or give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).?

File: app/views/entities/index.html.erb [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 </div>
 <% @entities.each do |entity| %>
-  <div class="flex justify-between items-center">
-    <div class="flex items-center gap-3 my-3">
-      <span class="text-xl font-semibold"><%= entity.name %></span>
+  <div class="flex justify-between items-center py-3 px-5 my-2 mx-3 bg-white drop-shadow-md">
+    <div class="">
+      <span class=""><%= entity.name %></span>
       <span>$<%= entity.amount%></span>
     </div>
```
Comment (OLIPLICHE) [inline app/views/entities/index.html.erb:15]: # Hi can you follow this requirement below to complete your App? I advise you to work on the 3rd point below:

- [ ] "Add a new transaction" page

The user fills out a form to create a new transaction with:

- 1 name (mandatory)

- 2 amount (mandatory)

- 3 👉 categories (mandatory at least one) 👈 

- **Here is your App.** 





NB: The comment above is part of the project requirement.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @BranBayou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Recorded a video presenting the features of the project 👍 
- Has deployed the application 👍 
- The App contains categories and transaction pages 👍 
- Has Units and integration testing 👍 
- Test has passed 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @BranBayou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

APPRVED
## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Proper implementation of suggested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**